index.html

```diff
@@ -51,28 +51,22 @@ <h1>My Name</h1>
       <section id="about">
         <header><h2>About Me</h2></header>
         <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
+          Hi, I'm Software Developer, specializing in JavaScript.
+          Turning your problems into solutions is my passion.
         </p>
       </section>
       <section id="projects">
         <header><h2>Projects Showcase</h2></header>
         <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
+          <a href="https://github.com/munozirianni1988/PersonalPageCYF"> Personal Page</a> <br>
```

Comment:
Instead of a p with a break tag, can you think of another html element used to show a list?